l2geth/consensus/clique/scheduler.go

```diff
@@ -134,6 +134,12 @@ func (schedulerInst *Scheduler) schedulerRoutine() {
 			Signature:    mockSignature,
 		}
 
+		// index block and sign L1toL2Tx here?
```

Comment:
Should signing happen here or in the `scheduler` package? i.e. `scheduler/drivers/scheduler/driver.go` etc